lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -288,9 +302,6 @@ PullImportPaths pullImport(
         return PullImportPaths(addedDagPaths, pulledUfePaths);
     }
 
-    // Record all USD modifications in an undo block and item.
-    UsdUndoBlock undoBlock(&UsdUndoableItemUndoItem::create("Pull import USD data modifications"));
```

Comment:
This is the redundant UsdUndoBlock that was improperly capturing what the import was doing. The importer has its own undo/redo capture system.